pxr/usdImaging/usdviewq/stageView.py

```diff
@@ -838,14 +847,18 @@ def rendererAovName(self):
     def __init__(self, parent=None, dataModel=None, printTiming=False):
         # Note: The default format *disables* the alpha component and so the
         # default backbuffer uses GL_RGB.
-        glFormat = QtOpenGL.QGLFormat()
+        glFormat = QGLFormat()
         msaa = os.getenv("USDVIEW_ENABLE_MSAA", "1")
-        if msaa == "1":
-            glFormat.setSampleBuffers(True)
-            glFormat.setSamples(4)
-        # XXX: for OSX (QT5 required)
-        # glFormat.setProfile(QtOpenGL.QGLFormat.CoreProfile)
-        super(StageView, self).__init__(glFormat, parent)
+        if PySideModule == 'PySide6':
+            if msaa == "1":
+                glFormat.setSamples(4)
+            super(StageView, self).__init__()
+            self.setFormat(glFormat)
+        else:
+            if msaa == "1":
+                glFormat.setSampleBuffers(True)
+                glFormat.setSamples(4)
+            super(StageView, self).__init__(glFormat, parent)
```

Comment:
Seems like a context object would also allow you to vary the timing of the call to super's `__init__`, for the two impl's here?  Naming is not as obvious to me, though...

Reply:
Moved this to an injected function `InitQtWidget`